cmd/boulder-va/main.go

```diff
@@ -89,6 +90,13 @@ func main() {
 
 	sbc := newGoogleSafeBrowsing(c.VA.GoogleSafeBrowsing)
```

Comment:
This should be in a `else` block after the feature check below as creating the client will cause a goroutine to be spawned in the background to update the local hash tree periodically which we probably want to avoid.